README.md

```diff
@@ -11,10 +13,24 @@
 - 📝 Line-specific comments on code changes
 - 🤖 Auto-fix suggestions with new PRs
 - 🎯 Project-specific guidelines
+- 🚫 Skip review functionality with special PR titles
 
 ## 🛠️ How to Use
 
-### 1. Set up Secrets
+### 1. Skip Code Review (Optional)
```

Comment:
🔍 Clear instructions on how to skip code review! This will help users understand the process better.